packages/container/tests/registry_test.js

```diff
@@ -489,6 +503,20 @@ QUnit.test('A registry can be created with a deprecated `resolver` function inst
   equal(registry.resolve('foo:bar'), 'foo:bar-resolved', '`resolve` still calls the deprecated function');
 });
 
+QUnit.test('A registry created with `resolver` function instead of an object throws assertion', function() {
+  expect(1);
+
+  let registry;
+
+  throws(() => {
```

Comment:
This should be swapped to use `expectAssertion` (so that it works properly in production builds since `Ember.assert` does not throw there).